libavcodec/vaapi_encode_h265.c

```diff
@@ -1209,14 +1207,14 @@ static av_cold int vaapi_encode_h265_get_encoder_caps(AVCodecContext *avctx)
 
     if (!priv->ctu_size) {
         priv->ctu_size     = 32;
-        priv->min_cb_size  = 16;
+        priv->min_cb_size  = 8;
```

Comment:
maybe this value doesn't work for the legacy driver.

Reply:
This change doesn't affact any setting to libva. Maybe the new fail may caused by changing surface align from 16 to 32. I will check it.